gno.land/pkg/gnoweb/gnoweb.go

```diff
@@ -109,6 +112,34 @@ func MakeApp(logger *slog.Logger, cfg Config) gotuna.App {
 	return app
 }
 
+var (
+	inlineCodePattern = regexp.MustCompile("`[^`]*`")
+	htmlTagPattern    = regexp.MustCompile(`<\/?\w+[^>]*?>`)
+)
+
+func checkHTMLInMarkdown(cfg *Config, content string) string {
```

Comment:
You can rename to sanitizeContent, a generic name that today will only filter HTML, but maybe more tomorrow.